source/includes/_queries.md

```diff
@@ -270,8 +267,60 @@ message GetAccountAssetTransactions {
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Account ID | account id to request transactions from  | `[a-z]{1,9}\@[a-z]{1,9}`
-Asset ID | asset id in order to filter transactions containing this asset | `[a-z]{1,9}\#[a-z]{1,9}`
+Account ID | account id to request transactions from  | username@domain
+Asset ID | asset id in order to filter transactions containing this asset | `[A-Za-z0-9]{1,9}\#[A-Za-z0-9]{1,9}`
```

Comment:
Maybe follow the similar way as for `Account ID`? e.g `asset_name@domain`

Reply:
maybe I should put description for asset_name and domain separately somewhere then